lib/readline.js

```diff
@@ -1331,7 +1331,8 @@ function codePointAt(str, index) {
   }
   return code;
 }
-exports.codePointAt = codePointAt;
+exports.codePointAt = util.deprecate(codePointAt,
+    'codePointAt() is deprecated. Use String.prototype.codePointAt');
```

Comment:
Yes people use the repl and other reads the code to find and use undocumented apis. I've done it myself, but I do not expect that code to work between versions.